### PR TITLE
Extra coverage

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

**Comment by ashtul** [comment]
Ping @gkorland

File: src/rebloom.c [hunk 1]
```diff
@@ -927,6 +927,7 @@ static void CFAofRewrite(RedisModuleIO *aof, RedisModuleString *key, void *obj)
     }
 }
 
+// LCOV_EXCL_START
```
Comment (gkorland) [inline src/rebloom.c]: please remove

File: src/cuckoo.c [hunk 2]
```diff
@@ -139,7 +139,7 @@ static size_t bucketCount(const CuckooBucket bucket, size_t len, CuckooFingerpri
 static size_t filterCount(const CuckooBucket *filter, const LookupParams *params) {
     const size_t ixs[2] = {params->i1, params->i2};
     if (params->i1 == params->i2) {
-        return bucketCount(filter[params->i1], CUCKOO_BKTSIZE, params->fp);
+        return bucketCount(filter[params->i1], CUCKOO_BKTSIZE, params->fp); // LCOV_EXCL_LINE probablistic
```
Comment (gkorland) [inline src/cuckoo.c]: please remove

File: src/rebloom.c (hunk 1 above)
Comment (ashtul) [reply src/rebloom.c]: done

File: src/cuckoo.c (hunk 2 above)
Comment (ashtul) [reply src/cuckoo.c]: done

**Pull request merged**